pages/Documents/ConversationalAI/Third-PartyBots/MicrosoftDirectLine/pva.md

```diff
@@ -0,0 +1,41 @@
+---
+pagename: PVA
+sitesection: Documents
+categoryname: "Conversational AI"
+documentname: Third-Party Bots
+subfoldername: Microsoft Direct Line
+permalink: third-party-bots-microsoft-direct-line-pva.html
+indicator:
+---
+
+### Power Virtual Agents - Overview
+
+[Power Virtual Agents](https://powervirtualagents.microsoft.com) is a platform provided by Microsoft for codeless 
+chatbot development.
+This platform therefore can be used instead of creating a bot with the Microsoft Bot Framework.
```

Comment:
`Thus, this platform can be used instead`

Reply:
rephrased the sentence now